src/main/java/school/faang/user_service/service/MentorshipService.java

```diff
@@ -0,0 +1,72 @@
+package school.faang.user_service.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import school.faang.user_service.dto.user.UserMentorshipDto;
+import school.faang.user_service.entity.User;
+import school.faang.user_service.mapper.UserMapper;
+import school.faang.user_service.repository.mentorship.MentorshipRepository;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class MentorshipService {
+    private final MentorshipRepository mentorshipRepository;
+    private final UserMapper userMapper;
+
+    public List<UserMentorshipDto> getMentees(long userId) {
+        log.info("Fetching mentees for mentor with ID: {}", userId);
+        User mentor = mentorshipRepository.findById(userId).orElse(null);
+        if (mentor == null || mentor.getMentees() == null) {
+            return List.of();
+        }
```

Comment:
Если у тебя нет такого пользователя в системе, то нужно возвращать ошибку
```suggestion
        User mentor = mentorshipRepository.findById(userId).orElseThrow(() -> new NoSuchElementException(...));
```

Reply:
"Если у пользователя нет менторов, то нужно вернуть пустой список" - прописано в условиях задачи в двух методах.

Reply:
В параметрах ты передаёшь userId - id пользователя у которого надо взять всех его учеников, а если этого пользователя нет, то надо выбрасывать ошибку. А твоё условие будешь дальше по коду, там стрим, если список будет пустой, то он вернёт его.